int64 instance
string acceptor_id
int64 proposal_num
string proposer_id
int64 accepted_value